src/Avalonia.Diagnostics/Diagnostics/Controls/FilterTextBox.axaml

```diff
@@ -0,0 +1,88 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:controls="using:Avalonia.Diagnostics.Controls"
+        x:CompileBindings="True"
+        x:DataType="controls:FilterTextBox">
+
+  <Design.PreviewWith>
```

Comment:
Can we actually preview devtools controls? Using `ControlCatalog.NetCore` does not work for me (missing control).

Reply:
No, we can't. But it worked to me with "public" modifier instead of "internal".